### PR TITLE
Public modals typo fixed

### LINKED ISSUE #8886: public table modals fonts are broken
https://team.carto.com/u/matallo/tables/tabula_fichero_4192_20160711_3_merge/public

screen shot 2016-07-14 at 15 04 36
screen shot 2016-07-14 at 15 04 40

also, download .carto

screen shot 2016-07-15 at 10 59 48

### LINKED ISSUE #8951: Export map from public page wrong typography
screen shot 2016-07-18 at 15 58 29

cc @xavijam

### LINKED ISSUE #8952: "OK, export" looks disabled (but it's not)
screen shot 2016-07-18 at 16 05 56

STR

1. Go to a public profile page.
2. Click export map.
3. Modal appears with "Ok, export" greyed out, marked as `is-disabled` (but it works).

cc @gfiorav @MariaCheca @xavijam 

PS: reported by @csobier

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #8886 
Fixes #8951 
Fixes #8952 

screen shot 2016-07-19 at 19 03 21
screen shot 2016-07-19 at 19 03 38
screen shot 2016-07-19 at 19 15 10
screen shot 2016-07-21 at 11 53 44

CR @javierarce

### CODE REVIEW

File: lib/assets/javascripts/cartodb/common/dialogs/export_map/templates/confirm.jst.ejs [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <span class="CDB-Button-Text CDB-Text is-semibold CDB-Size-medium u-upperCase">Cancel</span>
   </button>
 
-  <button class="js-ok CDB-Button CDB-Button--primary CDB-Button--big is-disabled">
+  <button class="js-ok CDB-Button CDB-Button--primary CDB-Button--big">
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/export_map/templates/confirm.jst.ejs]: I think there's a reason we had that `is-disabled` class there. Don't we have some piece of code that removes the class under certain conditions?

Comment (MariaCheca) [inline lib/assets/javascripts/cartodb/common/dialogs/export_map/templates/confirm.jst.ejs]: Actually that was a mistake caused by a copy&paste after the recent redesign, before that that button didn't have a disabled state :)

Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/export_map/templates/confirm.jst.ejs]: Ah, ok! :)

File: app/assets/stylesheets/common/option-card.css.scss [hunk 2]
```diff
@@ -47,6 +47,7 @@
   color: #2E3C43;
   background-color: #f9f9f9;
```
Comment (javierarce) [inline app/assets/stylesheets/common/option-card.css.scss]: I think you can use $cStructure-grayBkg in here.

File: app/assets/stylesheets/common/option-card.css.scss [hunk 3]
```diff
@@ -45,8 +45,9 @@
   max-height: 200px;
   text-align: left;
   color: #2E3C43;
-  background-color: #f9f9f9;
+  background-color: #F9F9F9;
```
Comment (javierarce) [inline app/assets/stylesheets/common/option-card.css.scss]: I think you can use `$cStructure-grayBkg` here.

**Comment by javierarce** [comment]
Check my last comment. The rest it's 👍

**Comment by MariaCheca** [comment]
✨

**Pull request merged**